Packs/CoreAlertFields/pack_metadata.json

```diff
@@ -6,7 +6,9 @@
     "author": "Cortex XSOAR",
     "url": "https://www.paloaltonetworks.com/cortex",
     "email": "",
-    "categories": [],
+    "categories": [
+        "Endpoint"
```

Comment:
Were is the pack page, are you sure "endpoint" suits it?

Reply:
Core is XDR